packages/hathor-rpc-handler/src/rpcMethods/getUtxos.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { HathorWallet } from '@hathor/wallet-lib';
+import { GetUtxosConfirmationResponse, GetUtxosRpcRequest, PromptHandler, UtxoDetails } from '../types';
+import { PromptRejectedError } from '../errors';
+import { ConfirmationPromptTypes } from '../types';
+import { validateNetwork } from '../helpers';
+
+/**
+ * Handles the 'htr_getUtxos' RPC request by prompting the user for confirmation
+ * and returning the UTXO details if confirmed.
+ * 
+ * @param rpcRequest - The RPC request object containing the method and parameters.
+ * @param wallet - The Hathor wallet instance used to get the UTXOs.
+ * @param promptHandler - The function to handle prompting the user for confirmation.
+ *
+ * @returns The UTXO details from the wallet if the user confirms.
+ *
+ * @throws {InvalidRpcMethod} If the RPC request method is not 'htr_getUtxos'.
+ * @throws {Error} If the method is not implemented in the wallet-service facade.
+ * @throws {PromptRejectedError} If the user rejects the prompt.
+ */
+export async function getUtxos(
+  rpcRequest: GetUtxosRpcRequest,
+  wallet: HathorWallet,
+  promptHandler: PromptHandler,
+) {
+  validateNetwork(wallet, rpcRequest.params.network);
+
+  const options = {
+    'token': rpcRequest.params.token,
+    'authorities': 0,
```

Comment:
Why don't you support authorities here as well?

Reply:
Added authorities, will need to update the RPC design